src/creator/tyler/Expression.java

```diff
@@ -0,0 +1,127 @@
+package creator.tyler;
+
+public interface Expression
+{
+    double getValue();
+}
+
+class Number implements Expression
+{
+    double value;
+
+    public Number(String value)
+    {
+        this.value = Double.valueOf(value);
+    }
+
+    @Override
+    public double getValue()
+    {
+        return value;
+    }
+}
+
+abstract class BinaryOperator implements Expression
+{
+    Expression left;
+    Expression right;
+
+    public BinaryOperator(Expression left)
+    {
+        this.left = left;
+    }
+
+    public Expression getLeft()
+    {
+        return left;
+    }
+
+    public Expression getRight()
+    {
+        return right;
+    }
+
+    public Expression setRight(Expression right)
+    {
+        this.right = right;
+        return this;
+    }
+}
+
+class Sum extends BinaryOperator
+{
+    public Sum(Expression left)
+    {
+        super(left);
+    }
+
+    @Override
+    public double getValue()
+    {
+        return left.getValue() + right.getValue();
+    }
+}
+
+
```

Comment:
Add Product, Difference, Quotient, etc. classes to represent each operator that you implement.